nautobot/extras/forms.py

```diff
@@ -1207,6 +1207,32 @@ class RelationshipAssociationFilterForm(BootstrapMixin, forms.Form):
     )
 
 
+class RelationshipAssociationModelFilterForm(forms.Form):
```

Comment:
It's also really looking like we could use a generic `NautobotFilterForm` class as well.